subsys/logging/log_output.c

```diff
@@ -310,25 +313,26 @@ static void hexdump_print(struct log_msg *msg,
 }
 
 static void raw_string_print(struct log_msg *msg,
-			     struct log_output_ctx *ctx)
+			     const struct log_output *log_output)
 {
-	assert(ctx->length);
+	assert(log_output->size);
 
 	size_t offset = 0;
 	size_t length;
 
 	do {
-		length = ctx->length;
-		log_msg_hexdump_data_get(msg, ctx->data, &length, offset);
+		length = log_output->size;
+		log_msg_hexdump_data_get(msg, log_output->buf, &length, offset);
```

Comment:
s/log_msg_hexdump_data_get/log_msg_raw_data_get/ ?
It is a bit confusing using hexdump to access string.

Reply:
added comment.